spinalcordtoolbox/deepseg/inference.py

```diff
@@ -118,7 +118,7 @@ def segment_non_ivadomed(path_model, model_type, input_filenames, threshold, kee
     device = torch.device("cuda" if use_gpu else "cpu")
 
     # load model from checkpoint
-    net = create_net(path_model, device)
+    net = create_net(path_model, model_type, device)
```

Comment:
It seems like there's been some confusion with how the monai/nnunet inference works:

- There are two different functions (`monai.create_nnunet_from_plans` and `nnunet.create_nnunet_from_plans`)
- Both functions get called from the same location in code -- we call one or the other based on the `model_type`.

So, you don't have to pass the `model_type` parameter to the function or anything like that. You can safely assume that when `model_type == 'monai'`, then `monai.create_nnunet_from_plans` will be called. (In other words, you can just directly edit the MONAI function to change the value of `unet_max_num_features`.)

Reply:
No worries! Thank you for the heads up. :heart:

Reply:
I updated the code again in commit https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4661/commits/f9b1d2e41ef0edff9ce73958d5d6a15f53555b07, this should work now!